docs/modules/operation/pages/provisioning/introduction.adoc

```diff
@@ -45,7 +45,7 @@ Using an import requisition, this mechanism directs {page-component-title} to ad
 No discovery process is used for finding more interfaces or services.
 
 IMPORTANT: An import requisition is an XML definition of node entities to be provisioned from an external source into {page-component-title}.
-See the http://xmlns.opennms.org/xsd/config/model-import[requisition schema] (XSD) for more information.
+See the https://raw.githubusercontent.com/OpenNMS/opennms/develop/opennms-provision/opennms-provision-persistence/src/main/resources/xsds/model-import.xsd[requisition schema] (XSD) for more information.
```

Comment:
My only comment with this link is that it is pinned to the `develop` branch, which is the next major version, and doesn't necessarily reflect the current release version or a past version a use may be viewing. Not sure I'd call that blocking as most XSDs rarely change,.

Reply:
Good point. This is temporary until someone fixes the issue (NMS-14149).